test/core/memory_grow.wast

```diff
@@ -0,0 +1,30 @@
+;;test that newly allocated memory (program start and memory.grow) is zeroed
+
+(module
+  (memory 0)
+  (func (export "grow") (param i32) (result i32)
+    (memory.grow (get_local 0))
+  )
+  (func $load (param $v i32) (result i32)
+    (i32.load (get_local $v))
+  )
+  (func (export "new_allocate_memory_zero") (result i32)
```

Comment:
How about generalising this function to take a start and end offset and check all bytes in-between? That way you could easily test initial memory and multiple grows of different size.

Reply:
Thanks for your advice, I will add two parameters that represent start and end.